src/constructs/loadbalancing/clb.ts

```diff
@@ -0,0 +1,32 @@
+import type { CfnLoadBalancer, LoadBalancerProps } from "@aws-cdk/aws-elasticloadbalancing";
+import { LoadBalancer } from "@aws-cdk/aws-elasticloadbalancing";
+import type { GuStack } from "../core";
+
+enum DeletionOverrideProperties {
+  SCHEME = "Scheme",
+}
+
+interface GuClassicLoadBalancerProps extends LoadBalancerProps {
+  overrideId?: boolean;
```

Comment:
I guess the reasons that we'd want to do this have been documented [here](https://github.com/guardian/cdk/discussions/111)? I wonder if we should add a link to that issue and/or some other short explanation in the code to help people who are migrating existing stacks?

Reply:
Good point! Although this is something we've done in a lot of constructs so I'm not sure if we want to link from everywhere. There's also a doc about [migrating existing stacks](https://github.com/guardian/cdk/blob/main/docs/003-migrating-existing-stacks.md) that might be a good place for it to live. 

We should probably have a wider think about documentation as most of the constructs aren't that accessible right now to those who haven't worked on the library and/or don't have much prior CDK knowledge.

Reply:
Ah great, thanks for sharing that - I agree that repeating this comment everywhere will be unhelpful! Feel free to disregard the original comment for now and we'll think about a more general solution for this later.